src/main/java/seedu/address/logic/commands/RemarkCommand.java

```diff
@@ -0,0 +1,91 @@
+package seedu.address.logic.commands;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_PATIENTS;
+
+import java.util.List;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.patient.Patient;
+import seedu.address.model.patient.Remark;
+
+/**
+ * Changes the remark of an existing patient in the address book.
+ */
+public class RemarkCommand extends Command {
+
+    public static final String MESSAGE_ADD_REMARK_SUCCESS = "Added remark to Patient: %1$s";
+    public static final String MESSAGE_DELETE_REMARK_SUCCESS = "Removed remark from Patient: %1$s";
+
+    public static final String COMMAND_WORD = "remark";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Edits the remark of the patient identified "
+            + "by the index number used in the last patient listing. "
+            + "Existing remark will be overwritten by the input.\n"
+            + "Parameters: INDEX (must be a positive integer) "
+            + "r/ [REMARK]\n"
+            + "Example: " + COMMAND_WORD + " 1 "
+            + "r/ Likes to swim.";
```

Comment:
I'm not sure about this, but usually there is no space between prefix (`r/`) and content (`Likes to swim`) right?

Reply:
Oh yes great spot! I'll make the necessary changes, though I believe that applying trim on our argument input will have also removed the trailing whitespace.